docs/ansible.utils.index_of_filter.rst

```diff
@@ -171,14 +171,14 @@ Examples
 
     #### Simple examples
 
-    - set_fact:
+    - ansible.builtin.set_fact:
```

Comment:
```suggestion
    - name: Define a list
      ansible.builtin.set_fact:
```
`- name: ` is highly recommended to use by the Ansible doc

I can see the other places below to fix similarly